### PR TITLE
Code review by Omedale

### PR DESCRIPTION
**Pull request by ajaps**
#### What does this PR do?
 - Code review by Omedale

#### Description of Task to be completed?
Code review by omedela

#### How should this be manually tested?
 - none

#### Any background context you want to provide?
 - none

#### What are the relevant pivotal tracker stories?
 - none

#### Screenshots (if appropriate)
 - none

#### Questions:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+*  Run the app
+  * `$ npm start`
+
+## Tests
+*  The tests were written using Supertest, Chai.
+*  The test coverage is generated by `gulp-istanbul` package
+*  To run tests, navigate to the project's root directory
+*  Run the following commands.
+  *  `$ npm test
+
```
Comment (omedale) [inline README.md:55]: Add FAQ and Limitations

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (seyi-adeleke) [inline .eslintrc:1]: You should use a maximum line length of 80 characters

File: server.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import expressValidator from 'express-validator';
+import path from 'path';
+import routes from './server/routes';
+
+dotenv.config();
+
+const server = express();
+
+server.use(express.static(path.join(__dirname, '../Documentation')));
+// server.use(express.static(path.resolve(`${__dirname}./public`)));
```
Comment (omedale) [inline server.js]: remove all commented codes

File: server/controllers/documents.js [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+   * @function updateDocument
+   * @param {object} request request
+   * @param {object} response response
+   * @return {object} object - information about the document updated
+   */
+const updateDocument = (request, response) => {
+  const validatDoc = helper.verifyDocUpdateParams(request);
+  if (validatDoc) {
+    return response.status(412).json({
+      message: validatDoc,
```
Comment (omedale) [inline server/controllers/documents.js]: Use precise variable names

File: server/controllers/role.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  .then((result) => {
+    const verifiedParams = result.mapped();
+    const noErrors = result.isEmpty();
+    if (!noErrors) {
+      return response.status(412).json({ message: verifiedParams });
+    }
+    const roleId = Number(request.params.id);
+    if (roleId === request.decoded.data.roleId) {
+      return response.status(403).json({
+        message: 'forbidden',
```
Comment (omedale) [inline server/controllers/role.js]: Use descriptive messages

**Pull request closed without merging**